app/src/test/java/com/concordia/mcga/factories/ShadowMainActivity.java

```diff
@@ -0,0 +1,18 @@
+package com.concordia.mcga.factories;
+
+import android.content.Context;
+
+import com.concordia.mcga.activities.MainActivity;
+
+import org.mockito.Mockito;
+import org.robolectric.annotation.Implementation;
+import org.robolectric.annotation.Implements;
+
+@Implements(MainActivity.class)
+public class ShadowMainActivity {
```

Comment:
You might want to put this in a separate package. I don't think its a good idea to put testing objects in packages with actual code. Maybe make a package just for shadows so we cal all use it.